tools/pip/setup.py

```diff
@@ -92,8 +89,7 @@ def main():
     )
 
     # clean up build detritus:
-    shutil.rmtree(build_dir_path)
-    shutil.rmtree(os.path.join(script_dir, "{}.egg-info".format(output_name)))
+    shutil.rmtree(os.path.join(script_dir, "pip-egg-info", "{}.egg-info".format(output_name)))
```

Comment:
I don't see where the original path is set. Is this added when building the egg?